src/components/DocumentProducer/EditableTableModal/helpers.ts

```diff
@@ -48,6 +53,16 @@ export const newValueFromEntry = (value: string | number, type: CellVariableType
         type: 'Select',
         optionValues: [value as number],
       };
+    case 'Image':
+      const newPhoto = (value as PhotoWithAttributesAndUrl[])[0];
+
+      return {
+        id: -1,
+        listPosition: 0,
+        type: 'Image',
+        caption: newPhoto.caption ?? '',
+        citation: newPhoto.citation ?? '',
+      };
```

Comment:
Right now the call to `setCellValue` is commented out, but this value will be returned as a new cell value for a new image. I'm not sure if this data is correct or if the step to update the cell value is needed.